modules/dr-restoring-cluster-state.adoc

```diff
@@ -304,3 +304,21 @@ etcd-ip-10-0-173-171.ec2.internal                2/2     Running     0
 ----
 
 Note that it might take several minutes after completing this procedure for all services to be restored. For example, authentication by using `oc login` might not immediately work until the OAuth server pods are restarted.
+
+[id="dr-scenario-2-restoring-cluster-state_storage_{context}"]
+= Issues and workarounds in restoring persistent storage state
+
+If your {product-title} uses persistent storage of any form, typically a state of the world is stored outside etcd. It could be a ElasticSearch
+cluster running in a Pod or a database running in a statefulset. When {product-title} is restored from etcd backup, we are also restoring status of those workloads in Openshift. But depending on how old the etcd snapshot is - that status may be invalid or outdated.
```

Comment:
"restored from the etcd backup"

I'd avoid starting a sentence with `But`. You could drop this word and have the sentence read - "The status may be invalid or outdated depending on how old the etcd snapshot is."

Reply:
- "Pod" should be lowercase ("pod").
- Change the sentence "When OCP is restored..." to "When restoring from an etcd backup, the status of the workloads in OCP is also restored."
- Change the sentence "But depending..." to "However, if the etcd snapshot is old, the status may be invalid or outdated."